services/learn-card-network/brain-service/test/profiles.spec.ts

```diff
@@ -1410,6 +1410,46 @@ describe('Profiles', () => {
 
             jest.useRealTimers();
         });
+
+        it('allows setting the expiration date', async () => {
+            jest.useFakeTimers().setSystemTime(new Date('02-06-2023'));
+
+            await userA.clients.fullAuth.profile.generateInvite({
+                challenge: 'nice',
+                expiration: new Date('02-06-2025').getTime(),
```

Comment:
```suggestion
                expiration: 60 * 60 * 24 * 7 * 52 *3, // 3 Years
```
This is set as a TTL, _not_ as the number of ms since the unix epoch! This test would still pass even if you later set the year to 2027!

Tbh this is probably worth putting in the OpenAPI docs and possibly even renaming the param to something like TTL (or even just changing it to accept ms since epoch like you are expecting!) and explicitly adding a test for!